src/global/constants.h

```diff
@@ -22,11 +22,15 @@
  */
 #ifndef LSST_QSERV_CONSTANTS_H
 #define LSST_QSERV_CONSTANTS_H
+
 /**
  * @brief  Global constants.
  *
  */
 
+// Sytem includes
+#include <chrono>
+
```

Comment:
Sorry, this is a bad idea to get this header included in this particular header file. All classes in Qserv will catch this include, even though only a few of those actually need this. Besides, we should not use our own headers as "packages" for including standard (or any other headers). Each header should define custom products only. 

The side effect of having the include here - is an extra dependency for all (a few hundred) Qserv classes and increased compilation time.